package.json

```diff
@@ -10,7 +10,7 @@
     "pretest": "npm run build",
     "test": "run-s test:*",
     "test:standard": "standard",
-    "test:mocha": "mocha test/*.js -r babel-register -r babel-polyfill --timeout 20000 --exit",
```

Comment:
There is no asynchronous processing in this library, so the timeout option should be its default value.